velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -584,7 +584,7 @@ class PlanBuilder {
 
   /// A convenience method to add a LocalPartitionNode with a single source (the
   /// current plan node) and hive bucket property.
-  PlanBuilder& localPartition(
+  PlanBuilder& localPartitionByBucket(
```

Comment:
We don't have any place to use localPartition with bucket property? Thanks!

Reply:
What do you mean? In TableWriteTest.cpp above, we use localPartitionByBucket

Reply:
I see. Only one place use it. Thanks!